docs/app/pages/components/components-sections/input-controls/input-dropdown/input-dropdown.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="row">
+    <div class="col-md-5">
+
+        <ux-input-dropdown
```

Comment:
I moved the button down, so that it is not covered by the drop-down.

The gap between display and drop-down comes from the select component. I can remove it, if you wish, but that would make these two components inconsistent.

In the neutral style, I don't care, but in the Micro Focus style, these need to be consistent to comply with the requirements from our UX team.

Reply:
Ah yes, you are correct, this is consistent with the select component, I think it was just the button behind it that made it look different. Thanks for moving the button, ignore the spacing issue.